doc/develop/microscope.rst

```diff
@@ -91,6 +91,38 @@ Detectors:
 System:
  * lens: Contains parameters concerning the parabolic mirror and the lens system.
 
+Roles of components found in the CRYO/ENZEL system:
+-----------------------------------------------------
+Emitters:
+ * e-beam: Electron beam of the SEM to scan the sample.
+ * light: Controls the excitation light of the FM.
+
+Actuator:
+ * ebeam-focus: Changes the focus position of the e-beam. It has one axis: z. 
+ * filter: Emission filter to select a specific wavelength band.
+ * stage: Moves the sample. It has 3 linear axes (x, y, z) and 2 rotational axes (rx, rz). stage component has the following metadata:
+    1. FAV_POS_DEACTIVE.
+    2. FAV_POS_ACTIVE.
+    3. FAV_POS_COATING.
+    4. FAV_POS_AREA.
+    5. ION_BEAM_TO_SAMPLE_ANGLE.
+ * focus: Changes the distance between the sample and the optical detectors. It has one axis: z. It has two metadata:
+    1. FAV_POS_DEACTIVE.
+    2. FAV_POS_ACTIVE. 
+ * align: Alignment actuator. It has 2 axes: x and y. It has also 2 metadata:
+    1. FAV_POS_ACTIVE.
+    2. FAV_POS_DEACTIVE.
+
+Detectors:
+ * Sample thermostat: Controls the temperature of the sample.
+ * Cooler controller: Controls the starting and stopping of the cooling process by changing the temperature setpoint of the cryo-stage.
```

Comment:
I checked the code for these, they don't subclass neither a detector nor actuator class, so let's just put them under system roles. 
You can check the abstract class methods for the hardware components in:
src/odemis/model/_components.py